doc/z-chap19.xml

```diff
@@ -13,4 +13,20 @@
     <#Include Label = "IteratorFromPickledFile">
   </Section>
 
+  <Section>
+    <Heading>Reading and writing multiplication tables to a file</Heading>
+    The functions <Ref Func = "ReadMultiplicationTable"/> and 
+    <Ref Func = "WriteMultiplicationTable"/> can be used to read or write,
+    respectively, multiplication tables to a file.
```

Comment:
I am deleting the next paragraph, as advised, so this is now the last one. Can I omit the <P/>?